src/vs/workbench/contrib/files/browser/explorerViewlet.ts

```diff
@@ -296,7 +296,7 @@ viewsRegistry.registerViewWelcomeContent(EmptyView.ID, {
 });
 
 viewsRegistry.registerViewWelcomeContent(EmptyView.ID, {
-	content: localize({ key: 'noFolderHelpWeb', comment: ['Please do not translate the word "commmand", it is part of our internal syntax which must not change'] },
```

Comment:
Could many comments here be deleted in fact? Because there is no `command` string in the actual copy.

Reply:
It's possible, but I don't want to do that for another feature area in this PR